release/models/local-routing/openconfig-local-routing.yang

```diff
@@ -144,6 +151,17 @@ module openconfig-local-routing {
         routing protocols.";
     }
 
+     leaf apply-tag-set {
+      type leafref {
+        path "/oc-rpol:routing-policy/oc-rpol:defined-sets/" +
+          "oc-rpol:tag-sets/oc-rpol:tag-set/oc-rpol:name";
+      }
+      description
+        "Apply all the tags in the defined tag-set to the route. These tags
+        can be used for filtering routes that are distributed to other
+        routing protocols.";
```

Comment:
Add note that all protocols may not support a tag-set.  For example, OSPF only supports a single tag.

Reply:
With this update, this PR would become ready for last call.  Please update if you want to move this forward.